src/spacefinder/liveblog-adverts.ts

```diff
@@ -175,23 +209,39 @@ export const init = (): Promise<void> => {
 		return Promise.resolve();
 	}
 
-	const isServerSideAdsMode =
-		window.guardian.config.tests?.serverSideLiveblogInlineAdsVariant ===
-		'variant';
-	if (isServerSideAdsMode) {
+	const isMobile = getCurrentBreakpoint() === 'mobile';
+
+	// Temporary measure to identify whether we're in server-side inline ad insertion mode.
+	// Client-side ad insertion functionality will be removed very soon.
+	const numMobileAdsInServerSideMode = document.querySelectorAll(
+		'.ad-slot-container.ad-slot-mobile',
+	).length;
+	IS_SERVER_SIDE_MODE = numMobileAdsInServerSideMode > 0;
```

Comment:
Can we pull this piece of decisioning out into its own function?

```typescript
// Temporary measure to identify whether we're in server-side inline ad insertion mode.
// Client-side ad insertion functionality will be removed very soon.
const isServerSideMode = () => {
	const numMobileAdsInServerSideMode = document.querySelectorAll(
		'.ad-slot-container.ad-slot-mobile',
	).length;
	return numMobileAdsInServerSideMode > 0;
};
```

Reply:
This check is only temporary, so I'd prefer to leave it as it is for now.

Reply:
Sure thing!